packages/manager/apps/public-cloud/src/components/walkMe/translations/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "walkme_title": "Nouveau",
+  "walkme_content_server": "<strong>Votre menu a évolué pour vous faire gagner du temps !</strong> Retrouvez notamment vos produits VPS, serveurs dédiés et Private Cloud",
+  "walkme_content_server_mobile": "<strong>Votre menu a changé !</strong> Le nouvel onglet « server » regroupe tous vos produits serveurs dédiés, private cloud etc..",
```

```suggestion
  "walkme_content_server_mobile": "<strong>Votre menu a changé !</strong> Le nouvel onglet « server » regroupe tous vos produits serveurs dédiés, Private Cloud, etc…",
```
Because it was written `Private Cloud` but maybe the texts are not definitive :)